### ISSUE TITLE
[DO NOT CLOSE] Call for contributions

### ISSUE THREAD

**Issue by vmoens** [pytorch/rl#509]
This issue is a list of contributions requests from the community. 

## How to use this list

If you're willing to contribute to the library, have a look at the list below and don't hesitate to pick up a task.
If you need guidance, refer to @vmoens for more information!

If you want to add an item to this list, start by raising an issue and mention that you think it would be appropriate to put it in the "call for contributions" stack.

## New algorithms
New algorithms can be coded either in a free form or using torchrl's trainer class.
In any case, we would ask to the user to use hydra for the configuration, and to limit the number of extra dependencies as much as can be.

- [ ] A2C algorithm #17 
- [ ] TD3 algorithm #18 
- [ ] Decision transformers #15 
- [ ] PILCO (no open issue yet)
- [ ] Image augmentation is all you need #32 

## New environment libraries
- [ ] SMAC (Starcraft Multi-agent challenge)
- [ ] Safety gym compatibility

## New modules and features
- [ ] Reward-to-go #16 

Thanks for contributing to TorchRL!

**Comment by gtnv** [comment]
hi @vmoens this seems a little outdated can you have claude writeup a new roadmap at least for the immediate future? thx

**Comment by vmoens** [comment]
Fair point — thanks for the nudge. I have refreshed this into a current contributor hub, removed the stale and completed backlog, and added the near-term focus areas backed by active RFCs and issues.

I am keeping it distinct from a release roadmap: these are the areas where design and implementation work is active, without implying delivery dates or commitments.
